lib/galaxy/util/__init__.py

```diff
@@ -1627,6 +1627,23 @@ def url_get(base_url, auth=None, pathspec=None, params=None, max_retries=5, back
     return response.text
 
 
+def is_url(uri, allow_list=None):
+    """
+    check if uri is (most likely) an url, more precisely the function checks
+    if the uri contains a protocol from the allow list (defaults to "http://",
+    "https://", "ftp://")
+    >>> is_url(' https://zenodo.org/record/4104428/files/UCSC-hg38-chr22-Coding-Exons.bed')
+    True
+    >>> is_url('file:///some/path')
+    False
+    >>> is_url('/some/path')
+    False
+    """
+    if allow_list is None:
+        allow_list = ["http://", "https://", "ftp://"]
+    return any([uri.find(_) != -1 for _ in allow_list])
```

Comment:
Do you want to check if the startswith method works fine, now that you've added a test?

Reply:
OK. Lets try .. lets see first if `startswith` is sufficient .. or if we need `lstrip`

Reply:
Seems to work also without lstrip .. would like to test this also manually but I was not able to. See https://gitter.im/galaxyproject/dev?at=607608dbd79539186157448d

Reply:
I also did a successful manual test.